README.md

```diff
@@ -131,21 +135,21 @@ While it is possible to use specific TTG backend explicitly, by using the approp
 
 ### Initialize
 
-To initialize TTG runtime invoke `ttg::initialize(argc, argv)`; there are several overloads of this function that also accept other optional parameters, such as the number of threads in the main thread pool, the MPI communicator for execution, etc.
+To initialize the TTG runtime invoke `ttg::initialize(argc, argv)`. There are several overloads of this function that also accept other optional parameters, such as the number of threads in the main thread pool, the MPI communicator for execution, etc.
 
 ## Specify a TTG
 
-To make a TTG create and connect one or more TTs. The simplest TTG consists of a single TT.
+To make a TTG, create and connect one or more TTs. The simplest TTG consists of a single TT as in the hello-world example.
 
-The "Hello, World!" example contains a single TT that executes a single task (hence, task ID can be omitted, i.e., void) that does not take and produce any data. The easiest way to make such a TT  is by wrapping a callable (e.g., a lambda) with `ttg::make_tt`:
+The "Hello, World!" example contains a single TT that executes a single task (hence, task ID can be omitted, i.e., `void`) that does not take and produce any data. The easiest way to make such a TT  is by wrapping a callable (e.g., a lambda) with `ttg::make_tt`:
```

Comment:
```suggestion
Since the TT in the "Hello, World!" example generates a single task, the task ID can be omitted (its type is `void`). This task also does not take or produce any data. The easiest way to make such a TT  is by wrapping a callable (e.g., a lambda) with `ttg::make_tt`:
```